community/kernel/src/main/java/org/neo4j/kernel/impl/util/Cursors.java

```diff
@@ -45,13 +43,31 @@ public boolean next()
         @Override
         public Object get()
         {
-            return null;
+            throw new IllegalStateException( "no elements" );
```

Comment:
We claim so here: https://github.com/neo4j/neo4j/blob/3.2/community/primitive-collections/src/main/java/org/neo4j/cursor/Cursor.java#L29